.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+# Golang CircleCI 2.1 configuration file
+#
+# Check https://circleci.com/docs/2.1/language-go/ for more details
+version: 2.1
+
+executors:
+  goexecutor:
+    working_directory: /go/src/github.com/signalfx/lambda-go
+    docker:
+      - image: golang:1.10-alpine
+
+commands:
+  goexecutor_setup:
+    steps:
+      - run: apk add --no-cache curl make git bash coreutils
+
+jobs:
+  lint:
+    executor: goexecutor
+    steps:
+      - checkout
```

Comment:
You need to put the checkout _after_ the `goexecutor_setup` step so that git will be installed.